doc/manual/manual.tex

```diff
@@ -1939,10 +1939,23 @@ \subsubsection{The isothermal compression approximation (ICA)}
 so we get
 \begin{equation}
 \label{eq:stokes-2-compressible}
-\nabla \cdot \textbf{u} = -\frac{1}{\rho} \frac{\partial \rho}{\partial p} \rho \textbf{g} \cdot \textbf{u}
+\nabla \cdot \textbf{u} = - \rho \frac{1}{\rho} \frac{\partial \rho}{\partial p} \textbf{g} \cdot \textbf{u}
```

Comment:
Is there a reason you are moving the /rho? It was in the same place as in the equation above, so I feel the old way was easier to read.